windy-feed/app/wind-feed.js

```diff
@@ -0,0 +1,148 @@
+const https   = require('https');
+const ethers  = require('ethers');
+const fs      = require('fs');
+
+const root                = 'iexec_out';
+const determinismFilePath = `${root}/determinism.iexec`;
+const callbackFilePath    = `${root}/callback.iexec`;
+const errorFilePath       = `${root}/error.iexec`;
+
+
+/*****************************************************************************
+ *                                   TOOLS                                   *
+ *****************************************************************************/
+const sleep = (ms) => {
+	return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+const cat = (path) => {
+	try { return fs.readFileSync(path).toString(); } catch (e) { return null; }
+}
+
+/*****************************************************************************
+ *                                  CONFIG                                   *
+ *****************************************************************************/
+
+// coin api key
+//const APIKEY = 'KXsaxepCiFoAlxKXkLzDO5sIbuLojtjH';
+const APIKEY1 = '4a71fa5207ec7e06042f7e63a647a80b';
+// const APIKEY = '69CC0AA9-1E4D-4E41-806F-8C3642729B88';
+// const APIKEY = 'D2C881D6-0BBF-4EFE-A572-AE6DB379D43E';
+// const APIKEY = 'FB7B2516-70A1-42D8-8702-292F29F19768';
+
+
+// random delay
+const WAIT_MIN = parseInt(process.env.WAIT_MIN) || 0; // in ms
+const WAIT_MAX = parseInt(process.env.WAIT_MAX) || 0; // in ms
+
+/*****************************************************************************
+ *                                 ARGUMENTS                                 *
+ *****************************************************************************/
+var [lat, long ] = process.argv.slice(2);
+
+
+/*****************************************************************************
+ *                                HTTP QUERY                                 *
+ *****************************************************************************/
+
+
+var lat = lat || 37.39;
+var long = long || -122.08;
+
+
+const url = "https://api.openweathermap.org/data/2.5/weather";
+const query1 = `${url}?lat=${lat}&lon=${long}&appid=${APIKEY1}`;
+
```

Comment:
This is good but not deterministic. I would suggest replacing lat/lon with lat/lon/time and use the endpoint:
`http://history.openweathermap.org/data/2.5/history/city?lat={lat}&lon={lon}&type=hour&start={start}&cnt={cnt}`

Reply:
@Amxx 
Thanks for your review. But historical data is not for free subscription . https://openweathermap.org/faq#error401
I don't know what to do next.